extensionsGallery.json

```diff
@@ -5080,15 +5080,76 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+				{
+				"extensionId": "100",
+				"extensionName": "ads-searcheverywhere",
+				"displayName": "Search Everywhere",
+				"shortDescription": "Search tables, views, stored procedures, functions by name or code.",
+				"publisher": {
+					"displayName": "MikhailT",
+					"publisherId": "MikhailT",
+					"publisherName": "MikhailT"
+				},
+				"versions": [
+					{
+						"version": "1.0.0",
+						"lastUpdated": "07/23/2023",
+						"assetUri": "",
+						"fallbackAssetUri": "fallbackAssetUri",
+						"files": [
+							{
+								"assetType": "Microsoft.SQLOps.DownloadPage",
+								"source": "https://github.com/MikhailProfile/SearchEverywhere/releases/download/ads/ads-searcheverywhere-1.0.0.vsix"
```

Comment:
other 3rd party extensions currently set this to a download page instead of vsix file.

Reply:
Updated